packages/lib/src/chip/Chip.tsx

```diff
@@ -85,11 +33,11 @@ const Chip = styled.div<{ margin: ChipPropsType["margin"]; disabled: ChipPropsTy
 `;
 
 const LabelContainer = styled.span<{ disabled: ChipPropsType["disabled"] }>`
-  font-size: ${(props) => props.theme.fontSize};
-  font-family: ${(props) => props.theme.fontFamily};
-  font-weight: ${(props) => props.theme.fontWeight};
-  font-style: ${(props) => props.theme.fontStyle};
-  color: ${(props) => (props.disabled ? props.theme.disabledFontColor : props.theme.fontColor)};
+  font-size: var(--typography-label-l);
+  font-family: var(--typography-font-family);
+  font-weight: var(--typography-label-regular);
+  font-style: var(--font-style-normal);
```

Comment:
`font-style: var(--font-style-normal);` is the default value in a span element, we can remove it!